src/main/scala/com/cognite/sdk/scala/v1/files.scala

```diff
@@ -37,15 +41,19 @@ final case class FileCreate(
     metadata: Option[Map[String, String]] = None,
     assetIds: Option[Seq[Long]] = None,
     dataSetId: Option[Long] = None
-)
```

Comment:
Shouldn't we have the new parameters (sourceCreatedTime, etc.) here too?

Reply:
Yes, we should! Good catch. :+1: